packages/dashboard/components/app-page.tsx

```diff
@@ -16,15 +16,105 @@ const drawerWidth = 240;
 
 export const AppPage = (props: any) => {
   const { title, children } = props;
+  const useAuth = props.method === 'photon';
+  let maybeAccessToken;
+  if (process.browser) {
+    maybeAccessToken = localStorage.getItem('sidetree.access_token');
+  }
+  const accessToken: any = maybeAccessToken
+    ? JSON.parse(maybeAccessToken)
+    : undefined;
+
+  const [isAuth, setIsAuth] = React.useState(useAuth && !accessToken);
   const [mobileOpen, setMobileOpen] = React.useState(false);
+  const [clientId, setClientId] = React.useState('');
+  const [clientSecret, setClientSecret] = React.useState('');
+  const [isInvalidCredentials, setIsInvalidCredentials] = React.useState(false);
   const handleDrawerToggle = () => {
     setMobileOpen(!mobileOpen);
   };
 
+  const style = {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 400,
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 4,
+  };
+
+  const authenticate = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const response = await fetch('/api/1.0/authenticate', {
+      method: 'POST',
+      body: JSON.stringify({ clientId, clientSecret }),
```

Comment:
this seems like a mistake to be in client side code.

Reply:
I think we probably want a "sign in as a user option here"... and leave the client credentials stuff to M2M scenarios.

Reply:
Trying to make app functional with m2m auth enabled on the api.  I could prompt for an access token but then it would be confusing to how they would generate it.